src/sections/home/Home.Events.tsx

```diff
@@ -15,21 +15,24 @@ const HomeEvents: React.FC<{ events: Array<EventObject> }> = ({ events }) => {
         {events && events.length > 0 ? (
           events.map((value, index) => {
             const timing = getDateTime(value);
+            const eventLink = "https://acmucsd.com/events/" + value.title.replace(/ /g, "-") + "-" + value.uuid;
             return (
-              <div className="home__events__grid__container__event" key={index}>
-                <img src={value.cover} alt={value.title} />
-                <h2>
-                  <span>{timing.date}</span>
-                  {timing.time}
-                </h2>
-                {isURL(value.location.trim()) ? (
-                  <a className="link" href={getAbsoluteURL(value.location.trim())}>
+              <Link href={eventLink}>
```

Comment:
Can we add styling to make the cursor switch to a pointer when the card is hovered to indicate that it's clickable?

If we add a CSS class name, there's a property for this `cursor: pointer`